### PR TITLE
#35 Add SettingsViewController and link to changelog

### PR DESCRIPTION
**Pull request by felixfoertsch**
#35 I added the SettingsViewController and linked to the changelog using the SFSafariViewController. I would appreciate feedback and additional directions. 🤓

### CODE REVIEW

File: Area51/Sources/SettingsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.row {
+        case 0:
+            let changelogURL = "https://github.com/kgellci/Area51/blob/master/CHANGELOG.md"
+            guard let url = URL.init(string: changelogURL) else {
```
Comment (kgellci) [inline Area51/Sources/SettingsViewController.swift]: Can just do URL(string:

Comment (felixfoertsch) [reply Area51/Sources/SettingsViewController.swift]: Oh, cool. Is the `guard let` with `fatalError()` correct here?

File: Area51/Sources/SettingsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+//
+//  SettingsViewController.swift
+//  Area51
+//
+//  Created by Felix Förtsch on 15.01.19.
```
[the remark this answers is not in the document]
Comment (felixfoertsch) [reply Area51/Sources/SettingsViewController.swift]: Check. This is the auto-generated stub from Xcode.

File: Area51/Sources/SettingsViewController.swift (hunk 1 above)
Comment (kgellci) [reply Area51/Sources/SettingsViewController.swift]: Not really needed but can be refactored later. I will add an extension for non optional URL from string to make this case easier.

File: Area51/Sources/SettingsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import UIKit
+import SafariServices
+
+class SettingsViewController: UITableViewController {
+
+    override func viewDidLoad() {
```
Comment (kgellci) [inline Area51/Sources/SettingsViewController.swift]: do not need this function if it only calls super

File: Area51/Sources/SettingsViewController.swift [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch indexPath.row {
+        case 0:
+            let changelogURL = "https://github.com/kgellci/Area51/blob/master/CHANGELOG.md"
+            guard let url = URL(string: changelogURL) else {
+                fatalError()
+            }
+            present(SFSafariViewController.init(url: url), animated: true)
```
Comment (kgellci) [inline Area51/Sources/SettingsViewController.swift]: SFSafariViewController(url instead of init(url

Comment (felixfoertsch) [reply Area51/Sources/SettingsViewController.swift]: Check.

File: Area51/Sources/SettingsViewController.swift (hunk 3 above)
Comment (felixfoertsch) [reply Area51/Sources/SettingsViewController.swift]: Okay! I think this was also auto-generated. I'll remove it.

**Pull request merged**